test/Container/ApplicationFactoryTest.php

```diff
@@ -27,6 +27,68 @@ public function setUp()
         $this->factory   = new ApplicationFactory();
     }
 
+    protected function getContainer($config = null, $router = null, $finalHandler = null, $emitter = null)
```

Comment:
While I saw too a helper or many helper methods are needed I disagree with this proposal because add too much logic.

The problem with add logic branches in a test case (if, for, ...) is difficult to trust is bugfree. What test method test getContainer works as expected?